package.json

```diff
@@ -15,7 +16,6 @@
     "babel-brunch": "~6.0.0",
     "uglify-js-brunch": "~2.0.1"
   },
-  "files": ["README.md", "LICENSE.md", "package.json", "priv/static/phoenix.js", "web/static/js/phoenix.js"],
```

Comment:
These are my thoughts as well. A single line in the config that we almost never need to update seems like a  win as far as what's being included?